inference-engine/src/vpu/myriad_plugin/myriad_metrics.cpp

```diff
@@ -114,6 +116,12 @@ const std::unordered_set<std::string>& MyriadMetrics::OptimizationCapabilities()
     return _optimizationCapabilities;
 }
 
+std::string MyriadMetrics::DeviceArchitecture() const {
+    // TODO: Task 49309. Return same architecture for devices which can share same cache
```

Comment:
From this comment, it is not clear what is required to resolve `TODO`
What this method should actually return?

Reply:
Thank you